index.html

```diff
@@ -22,17 +22,21 @@ <h1 class="name">Anne Hölscher</h1>
     </p>
     <br />
     <div class="button-group">
-      <a href="https://de.wikipedia.org/wiki/Anne_Bonny" class="button"
+      <a href="#about-me" class="button"
         >Über mich</a
       >
-      <a href="https://giphy.com/gifs/chill-relax-chilling-jeCSe7Qqc7NwKUzxH7" class="button">Lebenslauf</a>
+      <a href="#lebenslauf" class="button">Lebenslauf</a>
       <a href="https://giphy.com/gifs/v6aOjy0Qo1fIA" class="button"
-        >Katzen-GIF</a
+        >Lieblings-GIF</a
       >
     </div>
     <br />
     <hr />
-    <p>
+    <br />
+    <image class="strand" src="beach.jpg" alt="Strand">
```

Comment:
I would recommend you to work with the `section` element to group content that belongs together. See MDN for more information about the [section element](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section).